modules/library/main/src/main/java/org/geotools/styling/visitor/RescalingMode.java

```diff
@@ -76,7 +63,11 @@ public String rescaleToStringInternal(double scaleFactor, Measure measure) {
         @Override
         public String rescaleToStringInternal(double scaleFactor, Measure measure) {
             if (measure.isRealWorldUnit()) {
-                return String.valueOf(measure.value);
+                if(measure.isRealWorldUnitInPixelDefault()){                    
+                    return String.valueOf(measure.value) + UNIT_SYMBOLS.get(measure.uom);
+                }else{
+                    return String.valueOf(measure.value);
```

Comment:
Hi Jody, I would prefer the current solution because it is consistent with the results of its sister method rescaleToExpression(...). 
rescaleToExpression() has unit tests expecting a result without "px" unit. 
Agree?